test.js

```diff
@@ -276,6 +276,40 @@ test('DELETE a file', async (t) => {
 
   t.deepEqual(await dirResponse.json(), ['example.txt'], 'Only one file remains')
 })
+test.only('Clear the cached blobs of a file', async (t) => {
+  const created = await nextURL(t)
+
+  const formData = new FormData()
+  formData.append('file', new Blob([SAMPLE_CONTENT]), 'example.txt')
+  formData.append('file', new Blob([SAMPLE_CONTENT]), 'example2.txt')
+
+  const uploadedResponse = await fetch(created, {
+    method: 'put',
+    body: formData
+  })
+  await checkResponse(uploadedResponse, t)
+
+  const file2URL = new URL('/example2.txt', created)
+  const clearResponse = await fetch(file2URL, {
+    method: 'delete',
+    headers: {
+      'Cache-Control': 'no-store'
+    }
+  })
+
+  await checkResponse(clearResponse, t, 'Able to clear cache.')
+
+  const dirResponse = await fetch(created)
```

Comment:
Yeah testing this is kinda annoying. There's no direct API for it yet.

We'd need to use the hypercore.has api on the blobs hypercore.

https://github.com/holepunchto/hypercore#const-has--await-corehasstart-end

I think we can add a TODO for that and trust that .clear worked for now

Reply:
Commented out the last request.